app/src/main/res/anim/scale_animation_fragment.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android">
+        <scale
```

Comment:
when you only use a property change such as scale, you dont need to wrap it inside a set tag, you can keep it like
<scale 
   ... properties
/>